<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-PO3T4V3" failures="1" tests="22" name="testCases.LoginPopUpTest" time="176.474" errors="1" timestamp="2022-07-08T16:20:47 AMT" skipped="10">
  <testcase classname="testCases.LoginPopUpTest" name="setup" time="11.955">
    <error message="Reached error page: about:neterror?e=networkProtocolError&amp;u=https%3A//craftbet.com/home&amp;c=UTF-8&amp;d=Firefox%20has%20experienced%20a%20network%20protocol%20violation%20that%20cannot%20be%20repaired.
Build info: version: &#039;4.2.1&#039;, revision: &#039;ac4d0fdd4a&#039;
System info: host: &#039;DESKTOP-PO3T4V3&#039;, ip: &#039;10.50.17.112&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;17.0.2&#039;
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Command: [0756a71a-fac3-448f-9f8f-560fe13550a5, clickElement {id=c78e9c25-2a1d-46ba-99ff-528de38bedc2}]
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 101.0, moz:accessibilityChecks: false, moz:buildID: 20220526203855, moz:debuggerAddress: localhost:53188, moz:geckodriverVersion: 0.31.0, moz:headless: true, moz:processID: 30316, moz:profile: C:\Users\NERSES~1\AppData\L..., moz:shutdownTimeout: 60000, moz:useNonSpecCompliantPointerOrigin: false, moz:webdriverClick: true, pageLoadStrategy: normal, platformName: WINDOWS, platformVersion: 10.0, proxy: Proxy(), se:cdp: ws://localhost:53188/devtoo..., se:cdpVersion: 85.0, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Element: [[FirefoxDriver: firefox on WINDOWS (0756a71a-fac3-448f-9f8f-560fe13550a5)] -&gt; xpath: //span[text()= &#039;English&#039;]]
Session ID: 0756a71a-fac3-448f-9f8f-560fe13550a5" type="org.openqa.selenium.WebDriverException">
      <![CDATA[org.openqa.selenium.WebDriverException: Reached error page: about:neterror?e=networkProtocolError&u=https%3A//craftbet.com/home&c=UTF-8&d=Firefox%20has%20experienced%20a%20network%20protocol%20violation%20that%20cannot%20be%20repaired.
Build info: version: '4.2.1', revision: 'ac4d0fdd4a'
System info: host: 'DESKTOP-PO3T4V3', ip: '10.50.17.112', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.2'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Command: [0756a71a-fac3-448f-9f8f-560fe13550a5, clickElement {id=c78e9c25-2a1d-46ba-99ff-528de38bedc2}]
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 101.0, moz:accessibilityChecks: false, moz:buildID: 20220526203855, moz:debuggerAddress: localhost:53188, moz:geckodriverVersion: 0.31.0, moz:headless: true, moz:processID: 30316, moz:profile: C:\Users\NERSES~1\AppData\L..., moz:shutdownTimeout: 60000, moz:useNonSpecCompliantPointerOrigin: false, moz:webdriverClick: true, pageLoadStrategy: normal, platformName: WINDOWS, platformVersion: 10.0, proxy: Proxy(), se:cdp: ws://localhost:53188/devtoo..., se:cdpVersion: 85.0, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Element: [[FirefoxDriver: firefox on WINDOWS (0756a71a-fac3-448f-9f8f-560fe13550a5)] -> xpath: //span[text()= 'English']]
Session ID: 0756a71a-fac3-448f-9f8f-560fe13550a5
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:53)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:184)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:167)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:142)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:551)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:257)
at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:78)
at jdk.internal.reflect.GeneratedMethodAccessor10.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:52)
at jdk.proxy2/jdk.proxy2.$Proxy20.click(Unknown Source)
at pageObjects.BasePage.clickOnElementIfVisible(BasePage.java:85)
at pageObjects.CraftBet_Header_Page.selectEnglishLanguageFromDropDown(CraftBet_Header_Page.java:85)
at testCases.BaseTest.setup(BaseTest.java:68)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:361)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:296)
at org.testng.internal.invokers.TestInvoker.runConfigMethods(TestInvoker.java:816)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:590)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
    </error>
  </testcase> <!-- setup -->
  <system-out/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLabelsPresenceVerification_Test" time="11.260"/>
  <system-out/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLogoPresenceVerification_Test" time="7.871"/>
  <system-out/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpCloseButtonFunctionality_Test" time="8.507"/>
  <system-out/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpEyeButtonFunctionality_Test" time="9.660"/>
  <system-out/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpRememberMeLabelClickAbility_Test" time="10.432"/>
  <system-out/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpCheckBoxClickAbility_Test" time="11.621"/>
  <system-out/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpForgotPasswordLink_Test" time="9.024"/>
  <system-out/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpSignUpLink_Test" time="14.066"/>
  <system-out/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLogInPositiveTest" time="11.786"/>
  <system-out/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLogInNegativeTest" time="7.325">
    <skipped/>
  </testcase> <!-- loginPopUpLogInNegativeTest -->
  <system-out/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLogInNegativeTest" time="0.000">
    <skipped/>
  </testcase> <!-- loginPopUpLogInNegativeTest -->
  <system-out/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLogInNegativeTest" time="0.000">
    <skipped/>
  </testcase> <!-- loginPopUpLogInNegativeTest -->
  <system-out/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLogInNegativeTest" time="0.000">
    <skipped/>
  </testcase> <!-- loginPopUpLogInNegativeTest -->
  <system-out/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLogInNegativeTest" time="60.857"/>
  <system-out/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLogInNegativeTest" time="2.110">
    <failure message="expected [false] but found [true]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:976)
at org.testng.Assert.assertEquals(Assert.java:1010)
at testCases.LoginPopUpTest.loginPopUpLogInNegativeTest(LoginPopUpTest.java:191)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
    </failure>
  </testcase> <!-- loginPopUpLogInNegativeTest -->
  <system-out/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLogInNegativeTest" time="0.000">
    <skipped/>
  </testcase> <!-- loginPopUpLogInNegativeTest -->
  <system-out/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLogInNegativeTest" time="0.000">
    <skipped/>
  </testcase> <!-- loginPopUpLogInNegativeTest -->
  <system-out/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLogInNegativeTest" time="0.000">
    <skipped/>
  </testcase> <!-- loginPopUpLogInNegativeTest -->
  <system-out/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLogInNegativeTest" time="0.000">
    <skipped/>
  </testcase> <!-- loginPopUpLogInNegativeTest -->
  <system-out/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpErrorMessageValidationInvalidUsername" time="0.000">
    <skipped/>
  </testcase> <!-- loginPopUpErrorMessageValidationInvalidUsername -->
  <system-out/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpErrorMessageValidationInvalidPassword" time="0.000">
    <skipped/>
  </testcase> <!-- loginPopUpErrorMessageValidationInvalidPassword -->
  <system-out/>
</testsuite> <!-- testCases.LoginPopUpTest -->
